Makefile

```diff
@@ -5,7 +5,8 @@ LOCAL_BIN ?= $(PWD)/bin
 deployOnHub ?= false
 RELEASE_BRANCH ?= main
 OCM_API_COMMIT ?= $(shell awk '/open-cluster-management.io\/api/ {print $$2}' go.mod)
-DOCKER_URI ?= quay.io/stolostron
+CALLER_REPO ?= stolostron # default repo to pull from
+DOCKER_URI ?= quay.io/$(CALLER_REPO)
```

Comment:
The `DOCKER_URI` has to be slightly different because the upstream is `quay.io/open-cluster-management` while the GitHub repo is `open-cluster-management-io`.